docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -90,7 +90,7 @@ approach as shown in the following example.::
    >>> import matplotlib.pyplot as plt
    >>> from matplotlib.image import imread
    >>> hv = HelioviewerClient()  # doctest: +REMOTE_DATA
-   >>> file = hv.download_png('2099/01/01', 4.8, "[SDO,AIA,AIA,304,1,100]", x0=0, y0=0, width=512, height=512)  # doctest: +REMOTE_DATA
+   >>> file = hv.download_png('2099/01/01', 4.8, "[SDO,AIA,AIA,304,1,100]", False)  # doctest: +REMOTE_DATA
```

Comment:
Why is this False now? 

Also what was wrong with the image?

Reply:
Sorry which image.
And it is just a boolean parameter set to false. This field can also be left blank as by default I have fixed it's value to false. But just to show that we can also change it's value I set it to false to show the users that this parameter can be modified as well

Reply:
eventLabels is a required parameter for download_png function which is set to false by default. If set to true it will annotate each event marker with a text label so to show that we can set this parameter I used it as an example. It can be omitted as the value is false by default.

And I deleted the image and added a new image in it's place because of the segfault. But if you say it a falut with my local machine I will add those back

Reply:
eventLabels is a required parameter for download_png function which is set to false by default. If set to true it will annotate each event marker with a text label so to show that we can set this parameter I used it as an example. It can be omitted as the value is false by default.

And I deleted the image and added a new image in it's place because of the segfault. But if you say it a falut with my local machine I will add those back

Reply:
Ah that is the `eventLabels` bool. That's fine. If its set to false, I would just remove it from the example. 

I didn't see new image which is why I asked. But GitHub and files can be picky at times. 

What part was seg faulting? The submap operation? That shouldn't segfault and as far as I am aware, it works on our continuous integration.

Reply:
Yup the submap operation was a segfault and the core was dumped so I thought to remove those lines and show the image by just using the peek function. But if its a problem with my local machine I guess I will add those things back and then push the changes. Though there is still an error in the documentation which I am not able to solve and I am working on it

Reply:
Yeah, submap should work. I'm not aware of an issue with it right now. What is the issue with the documentation?

Reply:
The error because of which Travis test failed. I knew there is an error with the code. Actually the dictionary from which we extract the data is not so easy to manage as it contains different number of elements for a given key value

Reply:
You mean `ValueError: invalid literal for int() with base 10: 'N'`?

Reply:
Yup

Reply:
I guess there is a problem in the way we are accessing the values from a dictionary

Reply:
No. that is a bug with NumPy v1.15.3. You have to install another version for it to work.

Reply:
UNEXPECTED EXCEPTION: TypeError("'int' object is not subscriptable")
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test/lib/python3.7/doctest.py", line 1329, in __run
    compileflags, 1), test.globs)
  File "<doctest helioviewer.rst[3]>", line 5, in <module>
TypeError: 'int' object is not subscriptable
/tmp/sunpy-test-7f2l1lz3/lib/python3.7/site-packages/docs/guide/acquiring_data/helioviewer.rst:38: UnexpectedException
 

This is the error I am talking about

Reply:
Looks like you are trying to slice a integer somewhere. I would check to see what specific part is causing that and see if you can rewrite that part.

Reply:
Well the issue would be one of `params['nickname'], params['sourceId']`. Params probably is not a dictionary anymore.

Reply:
The problem is with measurements.items()
If we try printing that we get the same error

Reply:
Maybe it would be better to rewrite that example to avoid the for loop and the use of `.items.()`.

Clearly something has changed with whatever `measurements` is. You should check that.

Reply:
Sure I will work on that. I am not that good with dictionaries but I will solve that for sure

Reply:
If you still can't solve it, I can checkout the code and run it myself to see if I find the issue.

Reply:
Sure I'll let you know if i fail